packages/web-components/fast-foundation/docs/integrations/angular.md

```diff
@@ -87,6 +87,17 @@ export class AppComponent {
 }
 ```
 
+
+To allow NgModule to contain Non-Angular elements named, add  the code in your `app/app.module.ts` file contents with this:
+
+```ts 
+import {CUSTOM_ELEMENTS_SCHEMA  } from '@angular/core';
```

Comment:
the spaces are off inside the `{..}` brackets.  Prettier may catch, this though I'm not 100% positive